pkgs/tools/security/vaultwarden/webvault.nix

```diff
@@ -69,6 +69,6 @@ in buildNpmPackage rec {
     changelog = "https://github.com/dani-garcia/bw_web_builds/releases/tag/v${version}";
     platforms = platforms.all;
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ dotlambda msteen mic92 ];
+    inherit (vaultwarden.meta) maintainers;
```

Comment:
@dotlambda or do you want to be moved to the main package?

Reply:
Sure